### ISSUE TITLE
AttributeError: This QueryDict instance is immutable

### ISSUE THREAD

**Issue by jedie** [divio/django-filer#1036]
Replace a existing filer image under "Advanced" results in:
```
File ".../lib/python3.5/site-packages/filer/admin/imageadmin.py" in clean_subject_location
  68.         self._set_previous_subject_location(cleaned_data)

File ".../lib/python3.5/site-packages/filer/admin/imageadmin.py" in _set_previous_subject_location
  37.         self.data['subject_location'] = subject_location

File ".../lib/python3.5/site-packages/django/http/request.py" in __setitem__
  434.         self._assert_mutable()

File ".../lib/python3.5/site-packages/django/http/request.py" in _assert_mutable
  431.             raise AttributeError("This QueryDict instance is immutable")

Exception Type: AttributeError at /de/admin/filer/image/807/change/
Exception Value: This QueryDict instance is immutable
```

Django==1.11.10
django-cms==3.4.5
django-filer==1.3.0

**Comment by yakky** [comment]
I cannot reproduce this on master

**Comment by jedie** [comment]
Think it was the bug https://github.com/divio/django-cms/issues/6035 in django CMS and fixed with v3.4.6

**Comment by yakky** [comment]
Thanks for the update
